x/dao/keeper/msg_server_distribution_result.go

```diff
@@ -40,11 +40,16 @@ func (k msgServer) resolveStagedClaims(ctx sdk.Context, start int64, end int64)
 	}
 
 	popParticipants := make(map[string]uint64)
+	daoModuleAmt := uint64(0)
 
 	for _, challenge := range challenges {
-		challengerAmt, challengeeAmt := getAmountsForChallenge(challenge)
+		_, challengerAmt, challengeeAmt := util.GetPopReward(challenge.Height)
 		popParticipants[challenge.Challenger] += challengerAmt
-		popParticipants[challenge.Challengee] += challengeeAmt
+		if challenge.GetSuccess() {
+			popParticipants[challenge.Challengee] += challengeeAmt
+		} else {
+			daoModuleAmt += challengeeAmt
+		}
```

Comment:
This has to be removed. The handling of daoModule-related tokens needs to be completed and should be done explicitly for all cases and not at this stage (as we discussed before).

Reply:
done